README.md

```diff
@@ -1,10 +1,14 @@
 # vets-json-schema [![Build Status](https://travis-ci.org/department-of-veterans-affairs/vets-json-schema.svg?branch=master)](https://travis-ci.org/department-of-veterans-affairs/vets-json-schema)
 
+## Enviornment
+node v6.11.1
+
 ## I want to...
 
 | I want to... | Then you should... |
 | ------------ | ------------------ |
 | clone the site | `git clone https://github.com/department-of-veterans-affairs/vets-json-schema.git` followed by `cd vets-json-schema`, `npm install`. Run `npm install` anytime `package.json` changes. |
-| build the json schemas | `npm run build` |
+| build the json schemas and examples | `npm run build` |
+| build the json examples | `npm run build` |
```

Comment:
@jholton You're correct. This must-have slipped through the cracks. It should show first line:
`| build the json schemas and examples | `npm run build` |`

Reply:
The TODO's are still needed because we're waiting on the Business as to whether these fields need to be combined, or changed to enums. We expect these to change in a future PR.